aspnetcore/getting-started.md

```diff
@@ -24,20 +24,28 @@ uid: getting-started
    On macOS and Linux, open a terminal window. On Windows, open a command prompt.
 
    ```terminal
-   mkdir aspnetcoreapp
-   cd aspnetcoreapp
-   dotnet new web
+   dotnet new razor -o aspnetcoreapp
    ```
     
 4. Run the app.
 
-   The `dotnet run` command builds the app first if needed.
+   Change to the `aspnetcoreapp` directory and issue the `dotnet run` command. The `dotnet run` command builds the app first if needed.
```

Comment:
~Change to the `aspnetcoreapp` directory and issue the `dotnet run` command.~
I don't think most folks need that.